lua/core/utils/lsp.lua

```diff
@@ -215,21 +209,28 @@ astronvim.lsp.capabilities.textDocument.completion.completionItem.tagSupport = {
 astronvim.lsp.capabilities.textDocument.completion.completionItem.resolveSupport = {
   properties = { "documentation", "detail", "additionalTextEdits" },
 }
-astronvim.lsp.capabilities = user_plugin_opts("lsp.capabilities", astronvim.lsp.capabilities)
-astronvim.lsp.flags = user_plugin_opts "lsp.flags"
+astronvim.lsp.capabilities = user_opts("lsp.capabilities", astronvim.lsp.capabilities)
+astronvim.lsp.flags = user_opts "lsp.flags"
 
 --- Get the server settings for a given language server to be provided to the server's `setup()` call
 -- @param  server_name the name of the server
 -- @return the table of LSP options used when setting up the given language server
 function astronvim.lsp.server_settings(server_name)
```

Comment:
unsure, what your thoughts are on changing this to config since the directory changed as well?

```suggestion
function astronvim.lsp.config(server_name)
```

Reply:
ah yeah that's a good idea

Reply:
Changed in latest commit. Thanks for catching that @jay-babu !